lib/ansible/plugins/filter/to_json.yml

```
@@ -42,8 +43,8 @@ DOCUMENTATION:
     separators:
       description: The C(item) and C(key) separator to be used in the serialized output,
                    default may change depending on O(indent) and Python version.
-      default: "(', ', ': ')"
-      type: tuple
+      default: "[',', ': ']"
```

This value should be a list, not a string. The multi-line form should help clarify things:

```suggestion
      default:
        - ", "
        - ": "
```